default.nix

```diff
@@ -186,6 +188,43 @@ let
       };
     };
 
+    meadow = rec {
```

Comment:
I'm sure you're aware of the duplication here. Probably no need to do anything about it now unless there's something obvious.

Reply:
again, I think a refactoring is due once we know more about the direction of meadow.